apps/base-docs/tutorials/docs/1_verify-contract-with-basescan.md

```diff
@@ -0,0 +1,291 @@
+displayed_sidebar: null
+keywords: [Base, Solidity, ERC-721, token]
+difficulty: beginner
+tags: ['smart contracts']
+---
+
+[Basescan] is a block explorer specifically designed for [Base], offering developers a way to interact with and verify the smart contracts deployed on Base. Smart contract verification is a critical step in ensuring the transparency and security of onchain applications, as it allows others to review and validate the source code of deployed contracts. There are multiple ways to verify contracts and by the end of this tutorial you will learn how to verify a contract using the [Solidity] single file verification method using the [Basescan API].
+
+---
+
+## Objectives
+
+By the end of this tutorial, you should be able to:
+
+- Deploy a smart contract using [Foundry]
+- Interact with the [Basescan API] to verify your deployed contract
+- Obtain and configure a (free) Base RPC Node from [Coinbase Developer Platform (CDP)](https://portal.cdp.coinbase.com/products/node)
+
+---
+
+## Prerequisites
+
+**Familiarity with smart contract development and the Solidity programming language**
+
+Solidity is the primary programming language for writing smart contracts on Ethereum and Ethereum-compatible blockchains like Base. You should be comfortable with writing, compiling, and deploying basic smart contracts using Solidity.
+
+**Basic understanding of Foundry for Ethereum development**
+
+Foundry is a fast and portable toolkit for Ethereum application development. It simplifies the process of deploying, testing, and interacting with smart contracts. This tutorial assumes you have experience using Foundry to compile and deploy smart contracts.
+
+**Access to a Coinbase Developer Platform (CDP) account**
+
+The [Coinbase Developer Platform] provides access to tools and services necessary for blockchain development, including RPC nodes for different networks. You'll need to sign up for a CDP account to obtain a [Base RPC Node], which will be essential for deploying and interacting with your smart contracts on the Base blockchain.
+
+**Node + Basic API requests**
+
+## Jump Right In
+
+For this turotial, we will deploy a simple contract that is included in the Foundry quickstart. To do so, ensure that you have Foundry installed.
+
+If you don't have Foundry install it:
+
+```bash
+curl -L https://foundry.paradigm.xyz | bash
+```
+
+Once installed, create a Foundry project:
+
+```bash
+forge init verify_contracts
+```
+
+then change into the newly made directory:
+
+```bash
+cd verify_contracts
+```
+
+You should have a folder structure similar to this:
+
+```bash
+├── lib
+├── script
+├── src
+└── test
+```
+
+The `src` folder will contain a `Counter.sol` file which will serve as the country we want to deploy.
+
+:::note You will need ETH on Base to deploy
+You (the deployer wallet) will need some ETH in order to broadcast the transaction to the Base network. Fortunately, transactions are usually < 1 cent on Base mainnet.
```

Comment:
I'm 85% sure the admonishments need an empty line after the opening ::: and before the closing :::